packages/website/sidebars.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     {
       type: 'category',
       label: 'API',
-      items: ['api/api-reference'],
+      items: ['api/api-reference', 'api/equality-algorithm'],
```

Comment:
I'd personally move this to _Advanced_ section.

Reply:
Makes sense. I have no preference, so I'll move it.